CHANGES.txt

```diff
@@ -35,6 +35,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Drop remaining definitions of dict-like has_key methods, since
       Python 3 doesn't have a dictionary has_key (maintenance)
 
+  From Dillan Mills:
+    - Add support for the TARGET.relpath property 
```

Comment:
Should actually be `TARGET/SOURCE/TARGETS/SOURCES/CHANGED_TARGETS/CHANGED_SOURCES`